### DISCUSSION TITLE
Feature Request: Remove the X on the DB tabs, instead right click > close tab to avoid accidental closing of tabs

### DISCUSSION THREAD

**Discussion by locksec** [keepassxreboot/keepassxc#8877] [Ideas]
I have multiple tabs (KeePass databases) open during the day, but often when switching between them I accidentally close the tab by clicking the X. While other apps (like VS Code) also implement this, the tabs are wider so I have more room to click (avoiding accidental clicking on the X)

**Feature Request**: Remove X on the tabs and instead use Right Click > Close tab
**Current Workaround**: I have been working around this by changing the DB name and adding a bunch of whitespace after to make the tab wider.

**Comment by droidmonkey** [comment]
You can also use our keyboard shortcuts: Ctrl+PgUp/Down, Ctrl+TAB, and Ctrl+1/2/3/4...
